src/styles/card.scss

```diff
@@ -0,0 +1,98 @@
+
+.card {
+  display: flex;
+  justify-content: space-between;
+  border: 1px solid #f3f3f3;
+  flex-direction: column;
+  box-sizing: border-box;
+  border-radius: $border;
+  width: 200px;
+  background: $white;
+  box-shadow: 1px 1px 2px rgba(74, 102, 118, 0.3);
+  font-family: Roboto, Helvetica, sans-serif;
+  padding: 32px 16px 16px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    color: $title-color-value;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0;
+  }
+  &__desc {
+    color: $desc-color;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    margin-top: 4px;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+
+    &-value {
+      color: $title-color-value;
+      align-items: flex-end;
+      display: flex;
+      font-size: 10px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    align-items: center;
+    font-size: 16px;
+    margin-top: 24px;
+
+    &-title {
+      color: $desc-color;
+      font-size: 12px;
+      margin: 0;
+    }
+
+    &-value {
+      color: $title-color-value;
+      font-weight: 700;
+      margin-bottom: 0;
+      margin-top: 0;
+    }
+  }
+
+  &__button-link {
+    display: block;
+    box-sizing: border-box;
+    border: $btn-border;
+    border-radius: $border;
+    background-color: $btn-background;
+    text-align: center;
+    cursor: pointer;
+    text-transform: uppercase;
+    line-height: 37px;
+    font-size: 14px;
+    color: $white;
+    font-weight: 700;
+    text-decoration: none;
+    transition: all 1s;
```

Comment:
You are changing `background-color` and `color` , add the transition to only this properties